test/fenv/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Makefile
+#
+# makefile: fenv_test
+#
+# Copyright (C) 2017-2024 Tactical Computing Laboratories, LLC
+# All Rights Reserved
+# contact@tactcomplabs.com
+#
+# See LICENSE in the top level directory for licensing details
+#
+
+CXX=${RVCXX}
+ARCH=rv64imafdc
+ABI=lp64d
+
+CXX_OPTS = -std=gnu++1z -I ../../common/include -I ../../common/syscalls
+
+ifneq (,$(findstring clang,$(CXX)))
+	CXX_OPTS += -ffp-model=strict
+else
+	CXX_OPTS += -fsignaling-nans
+endif
+
+CXX_OPTS += -frounding-math -ftrapping-math -Werror=double-promotion -Werror=conversion -ffp-contract=on
+
+buildtests: fenv_test.makefile
+	make -j4 -f fenv_test.makefile CXX="$(CXX)" CXX_OPTS="$(CXX_OPTS)" ARCH="$(ARCH)" ABI="$(ABI)" TONEAREST UPWARD DOWNWARD TOWARDZERO
+
+runtests: fenv_test.makefile
+	make -j4 -f fenv_test.makefile CXX="$(CXX)" CXX_OPTS="$(CXX_OPTS)" ARCH="$(ARCH)" ABI="$(ABI)" RUN_TONEAREST RUN_UPWARD RUN_DOWNWARD RUN_TOWARDZERO
+	echo "Simulation is complete"
```

Comment:
The `runtests` target, which is built by `run_fenv.sh`, runs the tests, and, only if they succeed, prints the success message `"Simulation is complete"`.